lighthouse-core/gather/driver.js

```diff
@@ -1477,6 +1492,32 @@ class Driver {
     }
   }
 
+  /**
+   * @param {string} url
+   * @param {(LH.IcuMessage | string)[]} LighthouseRunWarnings
+   */
+  async getImportantStorageWarning(url, LighthouseRunWarnings) {
+    const usageData = await this.sendCommand('Storage.getUsageAndQuota', {
+      origin: url,
+    });
+    /** @type {Object.<string, string>} */
```

Comment:
```suggestion
    /** @type {Record<string, string>} */
```
have you been doing Closure compiled code recently? :)

Reply:
That was the syntax I found [here](https://jsdoc.app/tags-type.html)